test/UnitTests/PlatformServices.Desktop.Unit.Tests/Services/DesktopThreadOperationsTests.cs

```diff
@@ -100,5 +105,33 @@ public void ExecuteWithAbortSafetyShouldCatchThreadAbortExceptionsAndResetAbort(
             Assert.AreEqual(typeof(TargetInvocationException), exception.GetType());
             Assert.AreEqual(typeof(ThreadAbortException), exception.InnerException.GetType());
         }
+
+        [TestMethod]
+        public void TokenCancelShouldAbortExecutingAction()
+        {
+            // setup
+            var cancellationTokenSource = new CancellationTokenSource();
+
+            // act
+            cancellationTokenSource.CancelAfter(100);
+            var result = this.asyncOperations.Execute(() => { Thread.Sleep(10000); }, 100000, cancellationTokenSource.Token);
+
+            // validate
```

Comment:
Shouldn't we validate for something True as well ?

Reply:
Done.